firmware_tools/ghidra/vxhunter_core.py

```diff
@@ -537,32 +530,38 @@ def find_loading_address(self):
             prefix_keyword = '\x00_' + key_word + '\x00'
             key_word = '\x00' + key_word + '\x00'
             if key_word in self._firmware is False and prefix_keyword in self._firmware is False:
-                self.logger.info("This firmware didn't contain function name")
+                self.logger.info("Firmware does not contain a function named {}".format(key_word))
                 return None
         try:
             key_function_index = self._firmware.index('\x00' + function_name_key_words[0] + '\x00')
+            self.logger.debug("key_function_index: {}".format(key_function_index))
         except Exception as err:
             # Handler _ prefix symbols
+            self.logger.exception("Exception occurred while loading key_function_index: {}. Continuing...".format(err))
             key_function_index = self._firmware.index('\x00_' + function_name_key_words[0] + '\x00')
+            self.logger.debug("key_function_index: {}".format(key_function_index))
 
         str_start_address, str_end_address = self.find_string_table_by_key_function_index(key_function_index)
         self.get_string_table(str_start_address, str_end_address)
+        import json
```

Comment:
Please remove this test code.

Reply:
Sorry about that -- I've removed the test code.